fuzzers/frida_libpng/Cargo.toml

```diff
@@ -21,11 +21,10 @@ num_cpus = "1.0"
 which = "4.1"
 
 [target.'cfg(unix)'.dependencies]
-libafl = { path = "../../libafl/", features = [ "std", "llmp_compression", "llmp_bind_public" ] } #,  "llmp_small_maps", "llmp_debug"]}
+libafl = { path = "../../libafl/", features = [ "std", "llmp_bind_public" ] } #,  "llmp_small_maps", "llmp_debug"]}
```

Comment:
Why did you get rid of compression?

Reply:
I think it slows things down :(

Reply:
Do you think, or did you test? The microbenchmarks looked convincing, and it'll use far less ram